packages/frontend/tailwind.config.js

```diff
@@ -61,9 +64,11 @@ module.exports = {
       red: {
         300: '#FA3A3A',
         500: '#C32806',
+        600: '#C7141433',
         700: '#D70000',
       },
       blue: {
+        200: 'rgba(150, 192, 247, 0.2)',
```

Comment:
when you specify bg-blue-200 you should add bg-opacity-20 instead of changing the colour